pyannote/core/annotation.py

```diff
@@ -348,6 +349,19 @@ def __contains__(self, included):
         """
         return included in self.get_timeline(copy=False)
 
+    def write_rttm(self, file: TextIO):
+        """Write annotation to "rttm" file
+        Parameters
+        ----------
+        file : file object
+        """
+        for segment, _, label in self.itertracks(yield_label=True):
+            line = (
+                f'SPEAKER {self.uri} 1 {segment.start:.3f} {segment.duration:.3f} '
```

Comment:
Please handle corner case where "uri" was not assigned (e.g. use <NA> instead)

Reply:
I think you forgot a word... what should I use instead of uri ?

Reply:
Oooops. Something like `<NA>`.

Reply:
ok, can you be more specific about PEP8 ? I thought I messed up the white-spaces but I'm not sure what's wrong, is it the line length in write_rttm ?